docs/snapshotting/snapshot-support.md

```diff
@@ -670,7 +667,7 @@ m6i won't work on an m6g instance.
 | **x86_64**   | 4.14                   | 5.10                      | Y          |
 | **all**      | 5.10                   | 4.14                      | N          |
 | **all**      | 5.10                   | 6.1                       | Y          |
-| **all**      | 6.1                    | 5.10                      | Y          |
+| **all**      | 6.1                    | 5.10                      | N          |
```

Comment:
The line `| **all**      | 5.10                   | 6.1                       | Y          |` should be updated as well as arm 5.10 -> 6.1 does not work

Reply:
Thanks! done.